File: myproject11/app/src/main/java/com/codewithamit/myproject11/MainActivity.java
package com.codewithamit.myproject11;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void button(View view){

        TextView txtHello = findViewById(R.id.textView);
        TextView txtHello2 = findViewById(R.id.textView2);
        TextView txtHello3 = findViewById(R.id.textView3);

        EditText editTextTextPersonName = findViewById(R.id.editTextTextPersonName);
        EditText editTextTextPersonName2 = findViewById(R.id.editTextTextPersonName2);
        EditText editTextTextPersonName3 = findViewById(R.id.editTextTextPersonName3);

        txtHello.setText(editTextTextPersonName.getText().toString());
        txtHello2.setText(editTextTextPersonName2.getText().toString());
        txtHello3.setText(editTextTextPersonName3.getText().toString());

    }
}

File: TodoListapp/app/src/main/java/com/example/todolistapp/MainActivity.kt
package com.example.todolistapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import com.example.todolistapp.db.MyDbHelper
import com.example.todolistapp.db.TodoTable
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    val todos = ArrayList<Todo>()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val todoAdapter = ArrayAdapter<Todo>(
            this,
            android.R.layout.simple_list_item_1,
            android.R.id.text1,
            todos
        )

        val db = MyDbHelper(this).writableDatabase

        fun refreshTodoList(){
            val todoList = TodoTable.getAllTodos(db)
            todos.clear()
            todos.addAll(todoList)
            todoAdapter.notifyDataSetChanged()
        }

        lvtodos.adapter = todoAdapter
        refreshTodoList()

        btnadd.setOnClickListener {
            val newtodo = Todo(etnewtodo.text.toString(), false)

            TodoTable.insertTodo(db,newtodo)
            refreshTodoList()
        }

    }


}
















